htdocs/compta/prelevement/class/bonprelevement.class.php

```diff
@@ -121,6 +121,26 @@ public function __construct($db)
 		$this->fetched = 0;
 	}
 
+	/**
+	*	Remove any illegal characters for SEPA
+	*
+	*	@param	string	$str			String to remove illegal chars
+	*	@return string   	       		Cleaned string
+	*
+	*/
+	public static function removeIllegalChars($str)
```

Comment:
For my information, which character did you met that make the xml not valid ?

Reply:
We received this error from the CIC: `Cet élément est invalide - La valeur '<valeur>' est invalide d'après son type de donnée 'Max70Text' - La vérification de la contrainte de pattern a échoué.
(Non conforme à la norme SEPA et/ou ISO20022)` when the string contained `&`, `"`, `<`, `_`,`Ç`.